examples/arm/README.md

```diff
@@ -0,0 +1,26 @@
+This directory contains an example of how to cross-compile to arm
+with `rules_haskell`.
+
+Build with
+```
+nix-shell --pure --run "cd examples/arm; bazel build //... --platforms=//:linux_aarch64"
```

Comment:
Perhaps add a symlink to the toplevel `shell.nix` to `examples/arm`. `examples` does the same thing.

Reply:
Done.